authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc

```diff
@@ -6,23 +6,21 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Manual mode is supported for Alibaba Cloud, Amazon Web Services (AWS), Microsoft Azure, Google Cloud Platform (GCP), IBM Cloud, and Nutanix.
+Manual mode is supported for Alibaba Cloud, Amazon Web Services (AWS), global Microsoft Azure, Microsoft Azure Stack Hub, Google Cloud Platform (GCP), IBM Cloud, and Nutanix.
 
 [id="manual-mode-classic_{context}"]
 == User-managed credentials
 
-In manual mode, a user manages cloud credentials instead of the Cloud Credential Operator (CCO). To use this mode, you must examine the `CredentialsRequest` CRs in the release image for the version of {product-title} that you are running or installing, create corresponding credentials in the underlying cloud provider, and create Kubernetes Secrets in the correct namespaces to satisfy all `CredentialsRequest` CRs for the cluster's cloud provider.
+In manual mode, a user manages cloud credentials instead of the Cloud Credential Operator (CCO). To use this mode, you must examine the `CredentialsRequest` CRs in the release image for the version of {product-title} that you are running or installing, create corresponding credentials in the underlying cloud provider, and create Kubernetes Secrets in the correct namespaces to satisfy all `CredentialsRequest` CRs for the cluster's cloud provider. Some platforms use the CCO utility (`ccoctl`) to ease this process during installation and updates.
```

Comment:
Purely a suggestion ease/facilitate?

Reply:
I think I had facilitate and Vale told me to use a simpler word :joy:  I prefer facilitate so I am going with your suggestion (I think it's a simple enough word in tech docs anyway, sorry Vale :stuck_out_tongue:)